chain/chain.go

```diff
@@ -361,3 +365,18 @@ func importChain(content []byte) (*Chain, error) {
 
 	return chain, nil
 }
+
+// GetGenesisAccountBalance returns balance for genesis account based on its address (expressed in weis).
+// If not found in provided allocations map, 0 is returned.
+func GetGenesisAccountBalance(address types.Address, allocations map[types.Address]*GenesisAccount) (*big.Int, error) {
+	if genesisAcc, ok := allocations[address]; ok {
+		return genesisAcc.Balance, nil
+	}
+
+	return big.NewInt(0), fmt.Errorf("genesis account %s is not found among genesis allocations", address)
```

Comment:
should it be `return nil, fmt.Errorf("genesis account %s is not found among genesis allocations", address)`?